packages/@aws-cdk/core/lib/asset-staging.ts

```diff
@@ -51,6 +51,15 @@ export class AssetStaging extends Construct {
    */
   public static readonly BUNDLING_OUTPUT_DIR = '/asset-output';
 
+  /**
+   * Clears the asset hash cache
+   */
+  public static clearAssetHashCache() {
+    this.assetHashCache = {};
+  }
+
+  private static assetHashCache: { [key: string]: string } = {};
```

Comment:
Please describe in a comment what this does. It caches source hashes based on asset configuration.

Maybe also call it `sourceHashCache`?

Reply:
The name `sourceHash` is deprecated in favor of `assetHash` so I propose to stick with `assetHashCache`.

Reply:
Heh, I missed that. Alrighty.